blueoil/cmd/main.py

```diff
@@ -73,25 +71,16 @@ def init(output):
     required=False,
 )
 def train(config, experiment_id=None):
-    if not experiment_id:
-        experiment_id = 'train_{:%Y%m%d%H%M%S}'.format(datetime.now())
-
-    run_train(config, experiment_id)
-
-    output_dir = os.environ.get('OUTPUT_DIR', 'saved')
-    experiment_dir = os.path.join(output_dir, experiment_id)
-    checkpoint = os.path.join(experiment_dir, 'checkpoints', 'checkpoint')
-    if not os.path.isfile(checkpoint):
-        click.echo('Checkpoints are not created in {}'.format(experiment_dir), err=True)
+    try:
```

Comment:
Why we need a `try-catch` for only `train` part?
How about `convert` and `predict`?

Reply:
I moved [this error handling operation](https://github.com/blue-oil/blueoil/pull/536/files#diff-1ce1d3673b3c8bd027df36f98c119f8aL84-L85) to [here](https://github.com/blue-oil/blueoil/pull/536/files#diff-0e36593a8447a90b2de26bc526b2255bR75-R76).

Yes, we should use `try-catch` for `convert` and `predict`.
but in this PR, I did not refactor `convert` and `predict`.

Reply:
Okay, I see. 👍